src/pages/index.tsx

```diff
@@ -390,38 +250,50 @@ const cardsOnlyProduct = [
       "/card-Sleeves/card-Sleeve_Teal.avif",
       "/card-Sleeves/card-Sleeve_Hazel.avif",
     ],
+    imageAlt: [
+      "/card-Sleeves/card-Sleeve-Black_Alt.avif",
+      "/card-Sleeves/card-Sleeve_Ocean_Alt.avif",
+      "/card-Sleeves/card-Sleeve_Gray_Alt.avif",
+      "/card-Sleeves/card-Sleeve_Teal_Alt.avif",
+      "/card-Sleeves/card-Sleeve_Hazel_Alt.avif",
+    ]
```

Comment:
Image alt being an array of images seems odd here. Isn't the alt suppose to be a human readable representation of what this image is if it fails to load?

Reply:
changed to showMoreImage